rewrite-core/src/main/java/org/openrewrite/internal/PropertyPlaceholderHelper.java

```diff
@@ -160,4 +170,13 @@ private static boolean substringMatch(CharSequence str, int index, CharSequence
         }
         return true;
     }
+
+    private static boolean isMavenBuiltInProperty(@NonNull final String originalPlaceholder) {
```

Comment:
We've already annotated the package; that means we likely don't have to repeat ourselves here.
https://github.com/openrewrite/rewrite/blob/86ae0f10e668b825fb1379258c3924f55980df97/rewrite-core/src/main/java/org/openrewrite/internal/package-info.java#L16-L19

```suggestion
    private static boolean isMavenBuiltInProperty(final String originalPlaceholder) {
```